### PR TITLE
Conforming calendar icon to global sizing

### PR DESCRIPTION
**Pull request by steveduffin**
## Description
Fix
[Calendar] Aligns calendar navigation sizing to global spacing for better compatibility with internal assets / icons. Provides alternative color prop for compatiblity with `fill:currentColor` on svgs in consuming app (see https://css-tricks.com/cascading-svg-fill-color/)

## Related Issue
#341 

## Motivation and Context
Aligning calendar across the sky estate

## How Has This Been Tested?
https://s.codepen.io/steveduffin/debug/MrYoQQ/jVkpogKXwOqA cross all modern desktop browsers, IE9 and iOS 11 

## Screenshots

Before:
screen shot 2017-12-13 at 14 49 14

After:

screen shot 2017-12-13 at 15 20 49

## Types of Changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Browser Support

- [x] All

## Checklist

- [ ] All **design and code** conforms to the [Design Contribution Guidelines](https://github.com/sky-uk/toolkit/blob/develop/CONTRIBUTING.md#design-contributions).
- [ ] New functions and mixins have appropriate tests.
- [ ] All new and existing tests passed.
- [ ] I have added instructions on how to test my changes.

### CODE REVIEW

**Comment by joebell93** [comment]
Would be good to test this in a few existing applications to avoid shenanigans

+ as highlighted by the label, this needs Mike's review

**Comment by steveduffin** [comment]
All the apps I've seen using calendar are using our internal react library. I've being testing this locally in that app and it's looking grand. Do you know of any others?

**Comment by steveduffin** [comment]
@joebell93 what is it about this fix that requires Mike's attention? Assuming alignment with the approved icon set is a given, is it just about pixel perfect sizing?

**Comment by joebell93** [comment]
@steveduffin more of a final check over, will nudge today.

File: packages/sky-toolkit-ui/components/_calendar.scss [hunk 1]
```diff
@@ -129,11 +129,14 @@ $calendar-navigation-modifiers: (
  * Calendar Navigation - Icon
  *
  * Applied directly to the SVG icon element
+  1. Cater for approach of using fill:currentColor on svgs in consuimg app
+  2. Retain backwards compatibility
  */
```
Comment (joebell93) [inline packages/sky-toolkit-ui/components/_calendar.scss]: The above needs to be changed to include asterisks:

```
  * Applied directly to the SVG icon element
  * 1. Cater for approach of using fill:currentColor on svgs in consuimg app
  * 2. Retain backwards compatibility
```

You can probably do this by amending the commit and force pushing the branch

**Comment by joebell93** [comment]
Once the above is done I think this is good to go 👍

**Comment by joebell93** [comment]
You pressed "update branch" didn't you Steve :'(

**Comment by joebell93** [comment]
🤷‍♂️

**Comment by joebell93** [comment]
@steveduffin want me to fix this for you?

**Comment by steveduffin** [comment]
Yes please @joebell93, I don't dare touch anything now

**Pull request merged**